src/main/java/com/epam/spring/hw3/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.spring.hw3.service.impl;
+
+import com.epam.spring.hw3.dto.OrderDTO;
+import com.epam.spring.hw3.dto.UserDTO;
+import com.epam.spring.hw3.model.Order;
+import com.epam.spring.hw3.model.User;
+import com.epam.spring.hw3.repository.OrderRepository;
+import com.epam.spring.hw3.service.OrderService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Component

```

Comment:
Use @Service annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers